src/deluge/model/instrument/midi_instrument.cpp

```diff
@@ -240,11 +227,15 @@ void MIDIInstrument::sendMonophonicExpressionEvent(int32_t whichExpressionDimens
 	}
 
 	case 1: {
-		// mono y expression is limited to positive values, double it to match range
```

Comment:
nit: `case CC_NUMBER_MOD_WHEEL:`

Reply:
In this case it's referring to the expression dimension and not the CC, but I think there's an enum for that too